src/domains/idpay-app/api/idpay_onboarding_workflow/openapi.onboarding.yml.tpl

```diff
@@ -0,0 +1,378 @@
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+  /initiative:
+    put:
+      tags:
+        - onboarding
+      summary: Check the initiative prerequisites
+      operationId: checkPrerequisites
+      requestBody:
+        description: Id of the iniziative
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OnboardingPutDTO'
+            example:
+              initiativeId: string
+      responses:
+        '200':
+          description: Check successful
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RequiredCriteriaDTO'
+              example:
+                pdndCriteria:
+                  - code: string
+                    description: string
+                    authority: string
+                selfDeclarationList:
+                  - code: string
+                    description: string
+        '202':
+          description: Accepted - Request Taken Over
+          content:
+            application/json: { }
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '401':
+          description: Authentication failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '403':
+          description: This enrolment is ended or suspended
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+  /consent:
```

Comment:
Why we are not using `/initiative/[id]/consent` instead of passing the initiativeId as parameter?

Reply:
The main reasons why we put the attribute initiativeId in the request body are the fact that it is only a partial identifier of the onboarding resource and, furthermore, in this way it doesn't appear in the access logs.

Reply:
For the logs part:
1) is the initiativeId a sensitive data?
2) if it is sensitive you have to also change the operation below "/initiativeId/status"